app/javascript/article-form/articleForm.jsx

```diff
@@ -269,6 +270,17 @@ export default class ArticleForm extends Component {
     });
   };
 
+  toggleModal = () => {
```

Comment:
I was debating about having an explicit close and display modal method, but after trying that, I'm understanding the logic here better. Looks good!